lib/inspec/runner.rb

```diff
@@ -84,7 +84,7 @@ def add_content(tests, libs, options = {})
       end
 
       # evaluate the test content
-      tests = Array(tests) unless tests.is_a? Array
```

Comment:
Array(tests) does magic. In case you give it a Hash, it will turn it into an Array and embed it.